build/vsts_build.yaml

```diff
@@ -235,7 +235,14 @@ phases:
       msbuildVersion: "16.0"
       configuration: "$(BuildConfiguration)"
       msbuildArguments: "/t:AfterBuild"
-    condition: " and(succeeded(),eq(variables['BuildRTM'], 'false')) "
+
+  - task: MSBuild@1
+    displayName: "Build Final NuGet.exe (via ILMerge)"
+    inputs:
+      solution: "src\\NuGet.Clients\\NuGet.CommandLine\\NuGet.CommandLine.csproj"
+      msbuildVersion: "16.0"
+      configuration: "$(BuildConfiguration)"
+      msbuildArguments: "/t:ILMergeNuGetExe /p:ExpectedLocalizedArtifactCount=13"
```

Comment:
- Should it be 15, in according to the languages that VS is localized?
- Can it be specified in a constant ?

Reply:
It should be 13. Yes, it mirrors how many .resources.dll files we have. English is in .dll. So it is 13.

Happy to constantize this if appropriate. Let's discuss a few alternate techniques to see how they feel.

Reply:
testing fix on CI now.